packages/xo-server/src/xapi-object-to-xo.mjs

```diff
@@ -862,6 +862,17 @@ const TRANSFORMS = {
       vm: link(obj, 'VM'),
     }
   },
+
+  pusb(obj) {
+    return {
+      type: 'PUSB',
+
+      description: obj.description,
+      host: link(obj, 'host'),
+      passthroughEnabled: obj.passthrough_enabled,
+      usbGroup: link(obj, 'USB_group'),
```

Comment:
AFAIK, USB_group aren't implemeted in this file

Reply:
Even if the `USB_group` is not yet implemented in `xo-server`, `link(obj, 'USB_group')` will return the `uuid` of the `USB_group.`

Since it doesn't raise any errors, do we really want to remove it?

@julien-f, an opinion?

Reply:
1. Only add this field if it's useful for us
2. If what we want is the UUID, then use the field directly (and the field name should reflect this), if what we want is an opaque identifier and if we may add `USB_group` objects in the feature then use `link`.

Reply:
the ref of an usb_group is mandatory to create the virtual usb passed through a VM .

Reply:
nevermind, @b-Nollet  implemented it in #7399